project/plugins.sbt

```diff
@@ -12,7 +12,6 @@ addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"           % "3.12.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"       % "2.2.0")
 addSbtPlugin("com.typesafe.play" % "sbt-plugin"               % "2.8.19")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-digest"               % "1.1.4")
-addSbtPlugin("org.irundaia.sbt"  % "sbt-sassify"              % "1.5.1")
```

Comment:
I don't think you can delete this. it compiles the scss into css

Reply:
Resolved